### PR TITLE
Fix/dockerfile

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

**Review by thiagoyeds — APPROVED** [review]

File: docker/docker.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+readonly REPOSITORY=ufcgsaps/catalog
+readonly USAGE="usage: docker.sh [-a] {build|push|publish|run} <TAG>"
+readonly MY_PATH=$(cd "$(dirname "${0}")" || { echo "For some reason, the path is not accessible"; exit 1; }; pwd )
```
Comment (thiagomanel) [inline docker/docker.sh]: remove this "For some reason". It does not help at all.

**Review by thiagomanel — COMMENTED** [review]
improve msgs

File: docker/docker.sh (hunk 1 above)
Comment (wesleymonte) [reply docker/docker.sh]: Done.

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**